packages/core/src/actions/invoke.ts

```diff
@@ -94,17 +94,16 @@ export function invoke<
           return;
         }
         ref._parent = interpreter; // TODO: fix
-        actorCtx.defer?.((state: AnyState) => {
+        actorCtx.defer(() => {
+          if (actorRef.status === ActorStatus.Stopped) {
+            return;
+          }
```

Comment:
IMHO it's much better to use the `actorRef` from the outer scope here instead of looking this up in the `state`. I don't currently have a failing test case for the old code (and maybe there is none to be created) but, on principle, it's way more error-prone to look this up by the key/id. 

The intention of this function is to start this concrete `actorRef` and not an `actorRef` that happens to have the same `id` at the time this deferred function gets finally called.

Reply:
I was looking it up on the `state` before to "emulate" `invokeStates` from `statesToInvoke` in the SCXML algorithm, but this is cleaner.